MoneyManager/MoneyManager/Helpers/RandomHelper.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+
+namespace MoneyManager.DataAccessLayer.Helpers
+{
+    public static class RandomHelper
+    {
+        public static int NextInt32(this Random rng)
```

Comment:
what was the main reason of adding such ext-methods?

Reply:
I have used it in the previous implementation of InitializerDB, now, in fact, I don't need it. Just forget to get rid of it. Thank you